### ISSUE TITLE
Did the docker service changed?

### ISSUE THREAD

**Issue by danfergo** [ruffsl/ros_docker_demos#3]
I was following the tutorial multicontainer and i when executed `docker service ls` I got 
`Error response from daemon: This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again.`

After I was able to create a swarm I executed `docker service ls` again and got an empty table (no services). After reading through the internet I got the idead that recently docker swarm and [docker service command changed](https://blog.alexellis.io/docker-swarm-mode-part1/) (?). Is It is suposed to not be working as mentioned in the tutorial?

Thanks,

**Comment by danfergo** [comment]
Also,
docker-compose command option `--x-networking` seems to do not exist any more. I updated the docker-compose file to 3.2 version and ran `docker-compose up -d` and worked fine.
```
version: "3.2"

services:
  master:
    build: .
    container_name: master
    command: roscore

  talker:
    build: .
    container_name: talker
    environment:
      - "ROS_HOSTNAME=talker"
      - "ROS_MASTER_URI=http://master:11311"
    command: rosrun roscpp_tutorials talker

  listener:
    build: .
    container_name: listener
    environment:
      - "ROS_HOSTNAME=listener"
      - "ROS_MASTER_URI=http://master:11311"
    command: rosrun roscpp_tutorials listener
```

**Comment by ruffsl** [comment]
Docker Compose has evolved considerably since I first made this repo. I should really refactor it sometime. In addition, it seems that docker swarm has also changed. I'm not yet completely familiar with the current direction of whole the docker project, its gotten larger that I can keep track of, but it seems there is a migration from Docker Swarm to Swarm Mode:
https://stackoverflow.com/questions/40039031/what-is-the-difference-between-docker-swarm-and-swarm-mode

Perhaps you might find the Swarm Mode simpler to work with than I found the original Swarm codebase.

If you'd send any PR with fixes or updates to reflect the modern ecosystem, I'd be happy to review and incorporate the feedback.
